### DISCUSSION TITLE
I want to be a mod or admin of the turkey community.

### DISCUSSION THREAD

**Discussion by Rapamunte** [IBAX-io/go-ibax#1811] [Q&A]
### Clear and concise description of the problem

your new project go language I want to be a part of you turkey committee is a very strong one and I want to help you assuming I live in turkey

### Suggested solution

your new project go language I want to be a part of you turkey committee is a very strong one and I want to help you assuming I live in turkey

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by ibaxio** [comment]
Thanks for your support.

Please contact [@KyleIBAX](https://t.me/IBAXKyle) at Telegram.
